src/components/Uploader/Uploader.test.tsx

```diff
@@ -0,0 +1,45 @@
+import "@testing-library/jest-dom/extend-expect"
+import React, { useState } from "react"
+import { fireEvent, render } from "../../tests/utils"
+
+import Uploader, { IUploaderProps } from "./Uploader"
+
+export const DefaultTemplate = (props: IUploaderProps) => {
```

Comment:
Do we need to export this? Seems like it's not being used outside this test file